app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int, is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            appetite = self.appetite
+            self.appetite = 0
```

Comment:
Setting `self.appetite` to 0 after feeding might not be the best approach if you want to retain the original appetite value for future use. Consider resetting `self.appetite` only if necessary for your application logic.